### PR TITLE
Updated CLI to use new task filter

### PR DESCRIPTION
**Pull request by KennethEnevoldsen**
Updated CLI to use new task filter. Also removed a few flags which seems to not be used.

### CODE REVIEW

File: mteb/cmd.py [hunk 1]
```diff
@@ -84,12 +85,6 @@ def main():
     parser.add_argument(
         "--device", type=int, default=None, help="Device to use for computation"
     )
-    parser.add_argument(
-        "--batch_size", type=int, default=32, help="Batch size for computation"
-    )
```
Comment (imenelydiaker) [inline mteb/cmd.py:89]: I think this one is important, it's supposed to be used by the encode function of Evaluators

Comment (KennethEnevoldsen) [reply mteb/cmd.py:89]: But atm I don't think it is used so I chose to remove it to avoid users thinking that it is used.

**Review by Muennighoff — APPROVED** [review]
Amazing!

**Review by imenelydiaker — APPROVED** [review]
LGTM, thank you! 🚀

**Pull request merged**